addon/components/audit-fields-view.js

```diff
@@ -34,4 +33,38 @@ export default FlexberryBaseComponent.extend({
     @type String
   */
   newValueColumnCaption: undefined,
+
+  /**
+    Audit fields optimized for showing in template.
+
+    @property auditFieldsForShow
+    @type Array
+  */
+  auditFieldsForShow: Ember.computed('auditFields', function() {
+    let auditFieldsForShow = [];
+
+    const originalAuditFields = this.get('auditFields');
+    if (!originalAuditFields || !Ember.isArray(originalAuditFields)) {
+      return auditFieldsForShow;
+    }
+
+    originalAuditFields.forEach(field => {
+        let fieldName =  field.field || field.Field;
```

Comment:
```suggestion
        const fieldName =  field.field || field.Field;
```
Или так не принято в технологии?